.github/workflows/pull-ex-buildx-images.yml

```diff
@@ -0,0 +1,135 @@
+name: pull-ex-buildx-images
+
+on:
+  pull_request_target:
+    types: [opened, edited, synchronize, reopened, ready_for_review]
+    paths:
+      - "images/**"
+      - ".github/workflows/image-builder.yml"
+  push:
+    branches:
+      - main
+    paths:
+      - "images/**"
+      - ".github/workflows/image-builder.yml"
+  workflow_dispatch: {}
+jobs:
+  # Build the base Alpine image
+  build-alpine:
+    uses: ./.github/workflows/image-builder.yml
+    with:
+      name: alpine
+      dockerfile: images/alpine/Dockerfile
+      context: .
+
+  # Build alpine/git image that depends on build-alpine
+  build-alpine-git:
+    needs: build-alpine
+    uses: ./.github/workflows/image-builder.yml
+    with:
+      name: alpine-git
+      dockerfile: images/alpine/git/Dockerfile
+      context: .
+
+  # Build alpine/git/gke-aws-auth image that depends on build-alpine-git
+  build-alpine-git-gke-aws-auth:
+    needs: build-alpine-git
+    uses: ./.github/workflows/image-builder.yml
+    with:
+      name: alpine-git-gke-aws-auth
+      dockerfile: images/alpine/git/gke-aws-auth/Dockerfile
+      context: .
+
+  # Build alpine/hadolint image that depends on build-alpine
+  build-alpine-hadolint:
+    needs: build-alpine
+    uses: ./.github/workflows/image-builder.yml
+    with:
+      name: alpine-hadolint
+      dockerfile: images/alpine/hadolint/Dockerfile
+      context: .
+
+  # Build alpine/shellcheck image that depends on build-alpine
+  build-alpine-shellcheck:
+    needs: build-alpine
+    uses: ./.github/workflows/image-builder.yml
+    with:
+      name: alpine-shellcheck
+      dockerfile: images/alpine/shellcheck/Dockerfile
+      context: .
```

Comment:
You can remove that jobs, this images will be not used anywhere after: #12019

Reply:
done.